vm/devices/storage/disk_nvme/nvme_driver/src/namespace.rs

```diff
@@ -579,3 +664,15 @@ fn nvm_cmd(opcode: nvm::NvmOpcode, nsid: u32) -> spec::Command {
         ..FromZeroes::new_zeroed()
     }
 }
+
+/// Save/restore NVMe namespace data.
+#[derive(Protobuf, Clone, Debug)]
+#[mesh(package = "underhill")]
```

Comment:
Let's use the "nvme_driver" package name.

Reply:
Also, as mentioned elsewhere in a previous review, all the saved states pieces need to be in `save_restore` modules so that it's easy to find the state.

And I'd prefer that we just have one top-level module for `nvme_driver`, so that we can see all the saved state together in one place.

Reply:
For all nvme_driver related, except for nvme_manager and fixed_pool, ok.

Reply:
I think this time I did not miss anything - marking as resolved.